lang/en.js

```diff
@@ -183,3 +183,4 @@ var language = {
     }
   }
 };
+module.exports = { language: language };
```

Comment:
It doesn't break because there is no more code below that line, but at that point the javascript engine throws this error and stops processing the file:

![image](https://user-images.githubusercontent.com/29712657/64706382-4bee9800-d4a9-11e9-971a-6dbd12bccf8b.png)

> However, I've tried replacing this with if (typeof module !== 'undefined' && module.exports) { global.language = language; } but get an output of undefined

Where do you get undefined?

Reply:
If I try and run ```node js-json.js``` the output `en.json` just contains the string `undefined`

Reply:
`global.language = language;` attaches the language object to the global object (which is visible by all node modules) and it doesn't return anything, so the return value of `require('../lang/en.js');` is undefined. You'll need to use the same as the original code (`module.exports = { language: language };`) inside the if statement.

Reply:
ahh okay. Thanks :)

Reply:
I think this is now fixed. I no longer get a console error.